src/pages/Home/index.tsx

```diff
@@ -43,7 +44,12 @@ const Homepage = ({ order, setOrder }: OrderProps): JSX.Element => {
     <>
       <img style={{ width: '24rem' }} src={imageLink} alt={name}></img>
       <p>{description}</p>
-      <BobaModalform milk={milk} setMilk={setMilk} />
+      <BobaModalform
```

Comment:
CamelCase `F` BobaModalform

Reply:
i changed this to BobaModalForm from earlier comment , did you want it to be bobaModalForm instead ?

Reply:
your form is small case.
Use BobaModalForm